dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ranger/RangerPageIterator.java

```diff
@@ -56,7 +63,12 @@ static Page create(int offset, int limit) {
   @Override
   protected T computeNext() {
     if (pageIterator == null || !pageIterator.hasNext() && !lastPage) {
-      List<T> entries = fetcher.apply(Page.create(offset, pageSize));
+      List<T> entries;
+      try {
+        entries = fetcher.apply(Page.create(offset, pageSize));
+      } catch (RangerException e) {
+        throw new RuntimeException("Failed to fetch data from Ranger", e);
```

Comment:
Nit: consider including `offset` and `pageSize` in the exception message

Reply:
Ok, done.